packages/desktop-client/src/components/budget/report/BudgetSummary.tsx

```diff
@@ -301,6 +303,8 @@ export const BudgetSummary = memo(function BudgetSummary({
     onToggleSummaryCollapse,
   } = useReport();
 
+  isGoalTemplatesEnabled = useFeatureFlag('goalTemplatesEnabled');
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is this line necessary? To me it look like we already get the value on line 297.

Reply:
It doesn't work without it.  I'll change this line to ` const isGoalTemplatesEnabled = useFeatureFlag('goalTemplatesEnabled');` and remove lines 293 and 297.  That appears to work for me.

Reply:
Cool, I'm ok with that.

I'm happy to approve the PR if you can solve the merge conflict.